app/Resources/DoctrineMigrations/Version20170818144244.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace Ilios\Migrations;
+
+use Doctrine\DBAL\Migrations\AbstractMigration;
+use Doctrine\DBAL\Schema\Schema;
+
+/**
+ * Activate all session types.
+ */
+class Version20170818144244 extends AbstractMigration
+{
+    /**
+     * @inheritdoc
+     */
+    public function up(Schema $schema)
+    {
+        $this->abortIf($this->connection->getDatabasePlatform()->getName() !== 'mysql', 'Migration can only be executed safely on \'mysql\'.');
+        $this->addSql('UPDATE session_type SET active = 1');
+
+    }
+
+    /**
+     * @inheritdoc
+     */
+    public function down(Schema $schema)
+    {
+        $this->abortIf($this->connection->getDatabasePlatform()->getName() !== 'mysql', 'Migration can only be executed safely on \'mysql\'.');
+        $this->addSql('UPDATE session_type SET active = 0');
```

Comment:
This seems dangerous. I think we should just leave this at whatever state it currently is.